src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,23 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="Unknown" xml:space="preserve">
     <value>&lt;Unknown&gt;</value>
   </data>
+  <data name="IntellisenseBuildFailedMessage" xml:space="preserve">
+    <value>Encountered some issues while loading the project. this will prevent us from providing some features such as full solution analysis for this project and any projects that depend on this project.</value>
+  </data>
+  <data name="IntellisenseBuildFailedTitle" xml:space="preserve">
+    <value>Project loading failed.</value>
+  </data>
+  <data name="IntellisenseBuildFailedDescription" xml:space="preserve">
+    <value>To see what caused the issue, please try below.
```

Comment:
@jasonmalinowski  which we dont have page yet? and I actually think bing search is better. bing gives way better than that link. based on my experience on current bing result of error list. FxCop use that direct link and compiler errors uses bing. bing usually (or at least every time so far for me) shows the direct link + other useful information at the top (within 5 results). FxCop always make me to do manual bing search again since that page most likely not enough.